scripts/run_rm.py

```diff
@@ -89,6 +89,13 @@ def get_args():
         choices=["eager", "sdpa", "flash_attention_2"],
         help="Attention implementation to use (default: None)",
     )
+    #### custom arguments added for general-preference/GPM-Llama-3.1-8B and general-preference/GPM-Gemma-2B
```

Comment:
I'd like to not add any args if possible, if just for one model. Given we already have a config structure.